students/anthony_mckeever/lesson3/exercise_3/list_lab.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+
+"""
+Programming In Python - Lesson 3 Exercise 3: List Lab
+Code Poet: Anthony McKeever
+Start Date: 07/30/2019
+End Date: 07/30/2019
+"""
+
+def get_fruit():
+    return input("Give me another fruit! > ")
+
+def print_fruit(fruits_list, msg="Current Fruit:"):
+    print(msg,  fruits_list, "\n")
+
+print("Series 1: Fun with Fruit!\n")
+
+fruits_list = ["Apples", "Pears", "Oranges", "Peaches"]
+print_fruit(fruits_list)
+
+fruits_list.append(get_fruit())
+print_fruit(fruits_list)
+
+fruit = input(f"Choose a number between 1 and {len(fruits_list)} > ")
+print(f"You chose {fruit}: {fruits_list[int(fruit) - 1]}")
+
+fruits_list = [get_fruit()] + fruits_list
+print_fruit(fruits_list)
+
+fruits_list.insert(0, get_fruit())
+print_fruit(fruits_list)
+
+for fruit in fruits_list:
+    if fruit[0] == "P":
```

Comment:
I'd use lower() or upper() to catch case sensitive user input

Reply:
That is a great point!  I'll be sure to catch that in the future. :)